OneSignalSDK/unittest/src/test/java/com/test/onesignal/TestHelpers.java

```diff
@@ -373,6 +398,49 @@ else if (type == Cursor.FIELD_TYPE_FLOAT)
       return iams;
    }
 
+   @WorkerThread
+   static void deleteOldInAppMessages() {
```

Comment:
This is duplicated, it exists in `com.onesignal.OSInAppMessageRepository` as well. Also I don't see this being called in the tests so I think we can remove this `deleteOldInAppMessages` method here.

Reply:
good catch, this was used before but I changed it. TY!